tests.py

```diff
@@ -445,15 +439,22 @@ def test_run_tests_failure(self, mock_new_popen, mock_fail, mock_fopen):
         mock_process.communicate.return_value = result_string, None
         mock_new_popen.return_value = mock_process
 
-        mock_file = mock.MagicMock()
-        mock_file_enterer = mock.MagicMock()
-        mock_file_enterer.__enter__.return_value = mock_file
-        mock_fopen.return_value = mock_file_enterer
+        # mock_file = mock.MagicMock()
+        # mock_file_enterer = mock.MagicMock()
+        # mock_file_enterer.__enter__.return_value = mock_file
+        # mock_fopen.return_value = mock_file_enterer
```

Comment:
Can you drop these lines? Otherwise LGTM.

Reply:
Done.